### PR TITLE
Long press to show text selection options

### PR DESCRIPTION
**Pull request by kumarpalsinh25**
- Long press on any title or description text to show Text Selection Options.
- Removed unwanted code of pin description from pin-page

1) Space Title & Description:

https://github.com/acterglobal/a3/assets/51526480/3fd35bed-0137-4a56-a473-f099069ceff9

2) Chat Title & Description:

https://github.com/acterglobal/a3/assets/51526480/703fefb9-669a-49ec-bfed-a32b0c3c8e74

3) Event Title & Description:

https://github.com/acterglobal/a3/assets/51526480/06827ab3-cc15-4e56-8583-b3edda6bd79c

4) Pin Title & Description:

https://github.com/acterglobal/a3/assets/51526480/0dd42662-9d44-47e1-bcb4-484d056853ab

5) Task Item Title & Description:

https://github.com/acterglobal/a3/assets/51526480/8b68b9ef-d96a-482b-a279-70768b416ec4

6) Task List Title & Description:

https://github.com/acterglobal/a3/assets/51526480/d105ed1b-63d0-4e49-957f-98b7d9df3541

### CODE REVIEW

File: app/lib/features/pins/widgets/pin_item.dart [hunk 1]
```diff
@@ -76,157 +58,56 @@ class _PinItemState extends ConsumerState<PinItem> {
...
-      _PinDescriptionWidgetConsumerState();
-}
-
-class _PinDescriptionWidgetConsumerState
-    extends ConsumerState<_PinDescriptionWidget> {
-  late EditorState textEditorState;
+  Widget _pinDescription() {
+    final description = widget.pin.content();
+    if (description == null) return const SizedBox.shrink();
+    final formattedBody = description.formattedBody();
```
Comment (gnunicorn) [inline app/lib/features/pins/widgets/pin_item.dart:70]: :+1:

File: app/lib/features/pins/widgets/pin_item.dart [hunk 2]
```diff
@@ -253,39 +134,30 @@ class _PinDescriptionWidgetConsumerState
...
+    final pinEdit = ref.watch(pinEditProvider(widget.pin));
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: TextFormField(
+        key: PinItem.linkFieldKey,
+        onTap: () async =>
+            !pinEdit.editMode ? await openLink(pinEdit.link, context) : null,
+        controller: _linkController,
+        readOnly: !pinEdit.editMode,
+        decoration: const InputDecoration(
```
Comment (gnunicorn) [inline app/lib/features/pins/widgets/pin_item.dart:149]: not quite sure I understand how editing the pin link works now ... but a single click just opens the link, right?

**Review by gnunicorn — APPROVED** [review]
lgtm

File: app/lib/features/pins/widgets/pin_item.dart (hunk 2 above)
Comment (kumarpalsinh25) [reply app/lib/features/pins/widgets/pin_item.dart:149]: Good catch, I missed it out completely. I need to manage it same as title and description. Like option from tap and menu item. I will do the require changes regarding the same.

Comment (kumarpalsinh25) [reply app/lib/features/pins/widgets/pin_item.dart]: @gnunicorn 
I have did required fixes on pin page to have full functional support of our new UX pattern. Check below commit and reference video for the same.

Commit:
https://github.com/acterglobal/a3/pull/1860/commits/3f719ca8909203087c7bd3de2e24215892ae7aeb

Video:

https://github.com/acterglobal/a3/assets/51526480/a6ae6399-e71c-4a10-b6cc-cdee37bc5dd7

**Pull request merged**